### DISCUSSION TITLE
a bit of a silly question, but how do i increase the thickness of the margin here?

### DISCUSSION THREAD

**Discussion by jaashuu** [soulhotel/FF-ULTIMA#238] [Q&A]
hi, i want to increase the margin between the tab and the sidebar bounding box through sidebery. i'm not too good with css and i tried combing through some files but i could not figure it out. help appreciated.
<img width="1470" alt="Screenshot 2024-12-30 at 8 02 44 PM" src="https://github.com/user-attachments/assets/0e536176-f15c-4c69-b7e0-e4ab76dfd338" />

**Comment by soulhotel** [comment]
Glad you were able to figure out the content margins, for the spacing there you can add some padding to the vertical tabs container.
```
#vertical-tabs {
  padding-right:  5px !important;
}
```
I recommend throwing all of your changes in the bottom of the `userChrome.css` in the chrome folder or link a seperate file, so that you can save/find your changes easily.

**Reply by jaashuu** [reply]
thank you for your speedy working solution!

**Comment by jaashuu** [comment]
sorry to bother again, how would i add this margin if sidebery was autohiding? the previous solution works fine for not autohiding, but the margin is applied to the collapsed tabs as well when autohide is on. i would like it to apply the padding during hover and transition. originally, i tried applying a 3px padding-right on #sidebar:hover in the autohide file, which worked only on hover, but not during the transition, where it extended back out to the edge. any solutions?

**Comment by soulhotel** [comment]
are you talking about the content margins?

Try these lines here:

https://github.com/soulhotel/FF-ULTIMA/blob/b326f81ec0baed95369c09c490ad4a9910bd8e97/theme/all-global-positioning.css#L419-L431

I am about to be to bed for work tmmrw, so I'll definitely get back to you when I can.

**Comment by jaashuu** [comment]
no worries, i'm in no rush. sorry if i didn't explain it clear enough, i'm talking about the same margins in the photo. i wanted to try to apply 3px margins right where the arrow is pointed but only when the sidebar is not hidden. so during hover or transition. if it's not possible i'll just stop being nitpicky and deal with the thin margin as is.

**Comment by soulhotel** [comment]
This is sidebery? If it is, you'd have to do that in the sidebery css editor
